DDSim/DDSim/Helper/ParticleHandler.py

```diff
@@ -13,6 +13,18 @@ def __init__( self ):
     self._printEndTracking = False
     self._printStartTracking = False
     self._minDistToParentVertex = 2.2e-14*mm
+    self._enableDetailedHitsAndParticleInfo = False
+    self._enableDetailedHitsAndParticleInfo_HELP = "print out lots of info about MCHistory"
+
+  @property
+  def enableDetailedHitsAndParticleInfo(self):
+    """Enamble lots of printout on simulated hits and MC-truth inforamation"""
```

Comment:
Enamble --> Enable, inforamation --> information
Unless of course you drop the property completely